docs/tutorials/get-tasks-for-a-user

```diff
@@ -0,0 +1,65 @@
+---
+layout: page
+---
+
+# Tutorial: Get tasks for a user
+
+In this tutorial, you learn how to  retrieve tasks
+associated with a specific user in the service.
+
+Expect this tutorial to take about 10 minutes to complete.
+
+## Before you start
+
+Make sure you've completed the [Before you start a tutorial](before-you-start-a-tutorial) topic on the development system you'll use for the tutorial.
+
+## Retrieve tasks for a specific user
+
+Retrieving tasks for a specific user from the service requires sending a `GET` request to the appropriate endpoint with the user's ID.
+
+To retrieve tasks for a specific user:
+
+1. Ensure your local service is running. If not, start it by navigating to the API directory and running the JSON server with the following command:
+
+    ```shell
+    cd <your-github-workspace>/to-do-service/api
+    json-server -w to-do-db-source.json
+    ```
+
+1. Open the Postman app on your desktop.
+1. Create a new request with the following values:
+    - **METHOD**: GET
+    - **URL**: `{{base_url}}/tasks?user_id=<user_id>`
```

Comment:
```suggestion
    - **URL**: `{{base_url}}/tasks?user_id=1`
```
You should use a specific `user_id` in the tutorial so they get a response that matches your documentation. You can then explain that they can change the `user_id` value as needed in future calls.

Also the double braces filter out the text between them. 
See `docs/tutorials/get-a-user-by-name.md` for examples of markup that makes them visible, again.